src/components/pager/js/Pager.js

```diff
@@ -493,7 +493,7 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
             }
         }],
         modelListeners: {
-            "": "{that}.events.onModelChange.fire({change}.value, {change}.oldValue, {that})"
+            "{fluid.pager}.model": "{that}.events.onModelChange.fire({change}.value, {change}.oldValue, {that})"
```

Comment:
This change doesn't look like it's needed. In fact it looks like it will produce the same outcome as the we are currently in the `fluid.pager` component. (See: http://docs.fluidproject.org/infusion/development/ChangeApplierAPI.html#model-path-references)